src/components/sbb-checkbox/sbb-checkbox.scss

```diff
@@ -28,8 +28,23 @@
   --sbb-checkbox-cursor: default;
 
   @include sbb.if-forced-colors {
-    --sbb-checkbox-color: GrayText;
-    --sbb-checkbox-label-color: GrayText;
+    --sbb-checkbox-selection-color: Canvas;
```

Comment:
Question: it is already set at line 16, does it need to be set again?

Reply:
Yes it has as the --sbb-checkbox-selection-color is redefined a few lines above.